internal-services/catalog/create-advisory-task.yaml

```diff
@@ -155,6 +155,9 @@ spec:
           /home/utils/apply_template.py -o $ADVISORY_FILEPATH --data "$DATA" \
             --template /home/templates/advisory.yaml.jinja
 
+          # Ensure the created advisory file passes the advisory schema
+          check-jsonschema --schemafile schema/advisory.json "$ADVISORY_FILEPATH"
```

Comment:
Where is the schema dir coming from? I don't see it being created anywhere and I don't think it exists in the utils image.

Reply:
It is in the gitlab repo that we clone on line 121

Reply:
ah, ok, thanks